### DISCUSSION TITLE
Thanks so much for taking on and continuing this project.

### DISCUSSION THREAD

**Discussion by WilsonBradley** [reloxx13/QuestAppLauncher_Assets#3] [General]
UPDATE:
So turns out, something is corrupt / wrong with my Quest 2 install of QuestAppLauncher. I verified that it is not standard behavior to have no icons at start.
I have removed data/cache from app, completely removed aaa.QuestAppLauncher folder  
android/data/aaa.QuestAppLauncher.App/
Uninstalled QuestAppLauncher

Any other suggestions?

![aaa QuestAppLauncher App-20210302-135147](https://user-images.githubusercontent.com/8173893/109707608-fa840e00-7b5f-11eb-9b79-462ccc3e23d3.jpg)
![aaa QuestAppLauncher App-20210302-135209](https://user-images.githubusercontent.com/8173893/109707620-fd7efe80-7b5f-11eb-8fac-10fbf64a16f2.jpg)
![aaa QuestAppLauncher App-20210209-174840](https://user-images.githubusercontent.com/8173893/109707627-01128580-7b60-11eb-89be-dc3bc829e333.jpg)

![image](https://user-images.githubusercontent.com/8173893/109708002-73836580-7b60-11eb-959b-596b692acaf6.png)

So the background and update function works.. 
BUT no icons..

config.json

{
  "gridSize": {
    "rows": 5,
    "cols": 5
  },
  "sortMode": "az",
  "show2D": true,
  "autoCategory": "top",
  "customCategory": "right",
  "autoUpdate": true,
  "background": "backgrounds/background.png",
  "downloadRepos": [
    {
      "repoUri": "tverona1/QuestAppLauncher_Assets/releases/latest",
      "type": "github"
    }
  ]
}

..

**Comment by begit** [comment]
Hello,
the apk is here https://github.com/tverona1/QuestAppLauncher/releases/tag/v0.11.1
You can find informations here https://github.com/tverona1/QuestAppLauncher#configjson-configuration-file to add categories and  icons. You find them here https://github.com/reloxx13/QuestAppLauncher_Assets/releases/tag/v2021.02.08 for copy them. You must change the  the config.json too.
PS Thanks a lot for this good work. My launcher is now like I want!

**Reply by WilsonBradley** [reply]
Sorry.. this is still not working on Quest 2..
No icons.. figured out the background function...

BUT no icons..

**Comment by reloxx13** [comment]
Sorry i did not get a notification.
Hope u found the questapplauncher @begit mentioned.

This is just an asset repo providing genres and some more banners.

**Comment by WilsonBradley** [comment]
Thank you so much for replying.

Just to set expectations, by default it's normal to not show any icons? The older official version didn't have this prob. so not sure if it's because it needs more attention or my setup.

**Comment by WilsonBradley** [comment]
So , I put all the assets within 
Android/data/aaa.QuestAppLauncher.App/files/

appnames_o_sidequest.json
appnames_quest.json
appnames_quest_genrefied.json
iconpack_o_sidequest.zip
iconpack_quest.zip

Still only the gear and blank, no icons.

**Comment by reloxx13** [comment]
> So , I put all the assets within 
> Android/data/aaa.QuestAppLauncher.App/files/
> 
> appnames_o_sidequest.json
> appnames_quest.json
> appnames_quest_genrefied.json
> iconpack_o_sidequest.zip
> iconpack_quest.zip
> 
> Still only the gear and blank, no icons.

Please read the readme. You have to edit the config.json and not copy files.

**Reply by WilsonBradley** [reply]
Thanks.. But that is the problem, the readme is not clear or straight forward.
I got the back ground to work  ;-)

I do appreciate the help.

**Comment by WilsonBradley** [comment]
Nope.. 

Put this into a file called config.json here android/data/aaa.QuestAppLauncher.App/files/
using sidequest. 
still no go.. I did a update in settings and looks like it downloaded icons, but still just blank screen with gear.

Format of config.json

{
    "gridSize": {
        "rows": 3,
        "cols": 4
    },
    "sortMode": "mostRecent",
    "show2D": true,
    "autoCategory": "top",
    "customCategory": "right"
    "autoUpdate": true,
    "background": "backgrounds/my_background_360.jpg",
    "downloadRepos": [
      {
        "repoUri": "tverona1/QuestAppLauncher_Assets/releases/latest",
        "type": "github"
      }
    ]
}
gridSize: Setting

**Reply by WilsonBradley** [reply]
I hope someday someone will explain this and I get it working..

Cheers..
